package.json

```diff
@@ -1,23 +1,44 @@
 {
-  "name": "@kaliber/library",
-  "version": "1.0.0",
+  "name": "@kaliber/sanity-plugin-duplicate",
+  "version": "0.0.1",
+  "description": "Adds a (usable) duplicate to sanity",
   "main": "index.js",
   "sideEffects": false,
   "publishConfig": {
     "access": "public"
   },
+  "scripts": {
+    "build": "sanipack build",
+    "watch": "sanipack build --watch",
+    "verify": "sanipack verify",
+    "prepublishOnly": "sanipack build && sanipack verify",
+    "prepare": "[ -f scripts/addPeerDependencies.js ] && node scripts/addPeerDependencies.js"
```

Comment:
Deze is niet meer nodig. (https://github.com/Kaliber/kaliber-sanity-template/commit/5d4e684d44c0564a42c64579512931a104e7c9c4)

Reply:
Ik denk dat 'watch' nog wel nodig is. De aanpassing in kaliber-sanity-template zorgt er niet voor dat hij wordt geprecompiled namelijk (bij mij in ieder geval), en werkt het dus niet.

Reply:
@HJSielcken Erik bedoelt volgensmij specifiek r. 15 (`prepare`), dat kun je fixen met dat stukje webpack config.

Reply:
Klopt, bedoelde alleen `prepare` :-D